.ci.yaml

```diff
@@ -1676,6 +1676,18 @@ targets:
       task_name: web_size__compile_test
     scheduler: luci
 
+  - name: Linux android views
+    recipe: flutter/android_views
+    bringup: true
+    properties:
+      dependencies: >-
+        [
+          {"dependency": "android_sdk"},
+          {"dependency": "android_avd", "version": "31"}
+        ]
+    timeout: 60
```

Comment:
Does this need 60 minutes?

Reply:
Probably not, but it also seems that every other test has the same 60min timeout.